### ISSUE TITLE
Use flutter Native window as transparent overlay window on desktop 

### ISSUE THREAD

**Issue by sudongg** [flutter/flutter#96732]
Hi，
In desktop application development, it is very common to render content to the window through the main window handle, and create a new transparent window as an overlay to draw the UI.

After looking at the flutter window example, I found that flutter uses SetParent to attach the flutter Native Window to the main window.
That way, can I render the content onto the main window handle and let the flutter Native Window draw the UI as a transparent overlay?

**Comment by maheshmnj** [comment]
There are similar issues filed for respective platform

windows: https://github.com/flutter/flutter/issues/71735
Linux: https://github.com/flutter/flutter/issues/66751
MacOs: https://github.com/flutter/flutter/issues/59969

Closing as a duplicate.

**Comment by sudongg** [comment]
> There are similar issues filed for respective platform
> 
> windows: #71735 Linux: #66751 MacOs: #59969
> 
> Closing as a duplicate.

I checked fluent_ Acrylic project. He uses `SetWindowCompositionAttribute` to realize the translucency of windows on windows platform. He can solve it #71735

But with this method, when I render the content to the main window, the translucent window still displays the content behind the main window rather than the content I want to render.

I used to use skia to draw the overlay window UI, Set the SKAlphaType of SKImageInfo to Premul,  and use WinAPI `UpdateLayeredWindow` in the overlay window to make the window translucent.

Translucent windows implemented using `UpdateLayeredWindow` can also be attached to the main window using SetParent.
and It can correctly display the content rendered in the main window.

Flitter also uses skia internally. flitter should be able to do it.
